classes/helper/HelperOptions.php

```diff
@@ -173,13 +173,16 @@ public function generateOptions($option_list)
 
                 // Fill values for all languages for all lang fields
                 if (substr($field['type'], -4) == 'Lang') {
+                    $field['value'] = array();
                     foreach ($languages as $language) {
                         if ($field['type'] == 'textLang') {
                             $value = Tools::getValue($key.'_'.$language['id_lang'], Configuration::get($key, $language['id_lang']));
                         } elseif ($field['type'] == 'textareaLang') {
                             $value = Configuration::get($key, $language['id_lang']);
                         } elseif ($field['type'] == 'selectLang') {
                             $value = Configuration::get($key, $language['id_lang']);
+                        } else {
+                            continue;
```

Comment:
Yep, I still got a fatal error on the class Hook because of an object used as a String.

I fixed it in this commit (from the PR you saw last week 😉).

See https://github.com/PrestaShop/PrestaShop/pull/6172/commits/62268fdf9acbad856db76e18d098be3c3f173c97